frontend/model/contracts/shared/types.js

```diff
@@ -67,6 +68,7 @@ export const messageType: any = objectMaybeOf({
   attachments: arrayOf(objectOf({
     name: string,
     mimeType: string,
+    size: optional(numberRange(1, CHAT_ATTACHMENT_SIZE_LIMIT)),
```

Comment:
Some observations:

- Not all messages have attachments, but all attachments have a file size, right? So I think maybe it makes sense to make the `arrayOf` on line 68 `optional`, and make this size here mandatory.
- Since `CHAT_ATTACHMENT_SIZE_LIMIT` could change in the future and even be an admin configurable value, let's make this value `Number.MAX_SAFE_INTEGER` instead.

Reply:
Right. updated.